diff --git a/ios/RNSScreen.mm b/ios/RNSScreen.mm
index 9b66012dc8db1e3af0cb5457f8dad86e7f2fe5e3..b39fa2fe9043675e9040ba940e52595dcfe70d03 100644
--- a/ios/RNSScreen.mm
+++ b/ios/RNSScreen.mm
@@ -1501,6 +1501,7 @@ - (void)viewWillDisappear:(BOOL)animated
 - (void)viewDidAppear:(BOOL)animated
 {
   [super viewDidAppear:animated];
+  [self.screenView updateContentScrollViewEdgeEffectsIfExists];
   if (!_isSwiping || _shouldNotify) {
     // we are going forward or dismissing without swipe
     // or successfully swiped back
